admin/acceptance.py

```diff
@@ -1254,6 +1254,10 @@ def parseArgs(self, *trial_args):
 }
 
 
+def eliot_logging_acceptance():
```

Comment:
We put docstrings on pretty much every function.

Also it's not clear to me from the name what this function does. There's no verb,
so I don't know what the subject is. "configure_eliot_for_acceptance" or something
would solve that.